packages/atlas/src/components/_buttons/Button/Button.tsx

```diff
@@ -64,10 +66,11 @@ export const Button = forwardRef<HTMLButtonElement, ButtonProps>((props, ref) =>
       disabled={disabled}
       aria-disabled={disabled}
       variant={variant}
-      iconOnly={!!iconOnly}
+      iconOnly={iconOnly}
       textOnly={!!_textOnly}
       size={size}
       data-badge={badge}
+      aria-label={ariaLabel || (typeof children === 'string' ? children : undefined)}
```

Comment:
Correct me if I'm wrong, but I don't think we need to set up `arial-label` if we already have text inside the button. Button with a text inside is already accessible.

Reply:
The problem is that the direct child of a button is `div` and it turns out that it needs to be a string to be accessible, otherwise we need to set `aria-label`.

Reply:
That's right, but also not true in our case, as we have a `<span>` element inside of the `<button>`. That's why Google Lighthouse yielded this as not accessible.

Reply:
I'm not sure why Lighthouse doesn't like it, because it's correct to have nested elements with text. And it should have an accessible name. Try to run the lighthouse here: https://acrobaticregulareyestrain.drillprop.repl.co/ 

Anyway, let's keep `aria-label` here.